CHANGELOG.md

```diff
@@ -4,16 +4,67 @@
 
 ##### Security
 
+#### Loki
+
+##### Enhancements
+* [10416](https://github.com/grafana/loki/pull/10416) **lpugoy**: Lambda-Promtail: Add support for WAF logs in S3
```

Comment:
Should this go underneath `Promtail`?

Reply:
agree. Just copied it from "unleased" as is. Will update it. 

Would be great if we can enforce this when adding to the changelog entry by the author in the first place.

Reply:
Done 👍